fdbserver/SimKmsConnector.actor.cpp

```diff
@@ -42,21 +43,24 @@ struct SimEncryptKeyCtx {
 	EncryptCipherBaseKeyId id;
 	SimEncryptKey key;
 
-	explicit SimEncryptKeyCtx(EncryptCipherBaseKeyId kId, const char* data) : id(kId), key(data) {}
+	explicit SimEncryptKeyCtx(EncryptCipherBaseKeyId kId, const char* data) : id(kId), key(data, AES_256_KEY_LENGTH) {}
 };
 
 struct SimKmsConnectorContext {
 	uint32_t maxEncryptionKeys;
 	std::unordered_map<EncryptCipherBaseKeyId, std::unique_ptr<SimEncryptKeyCtx>> simEncryptKeyStore;
 
 	explicit SimKmsConnectorContext(uint32_t keyCount) : maxEncryptionKeys(keyCount) {
-		uint8_t buffer[AES_256_KEY_LENGTH];
+		const unsigned char SHA_KEY[] = "0c39e7906db6d51ac0573d328ce1b6be";
```

Comment:
Do we intend to checkin this code?

Reply:
This is not debug code. It is intent to make the encrypt keys deterministic after restart, otherwise SimKmsConnector will return different set of cipher keys after a crash in simulation.